src/bundle/Resources/views/content/tab/policies/limitation_values.html.twig

```diff
@@ -0,0 +1,124 @@
+{# fallback block for limitations without defined value mapper #}
+{% block ez_limitation_value_fallback %}
```

Comment:
Do you add anything particular to this template? Extending the original file from repository-forms would make more sense because it will make it safe after something was added there.